packages/wagmi/src/utils/defaultWagmiCoreConfig.ts

```diff
@@ -47,7 +47,7 @@ export function defaultWagmiConfig({
 
   // Enabled by default
   if (enableWalletConnect !== false) {
-    connectors.push(alphaWalletConnect({ projectId, metadata, showQrModal: false }))
+    connectors.push(walletConnect({ projectId, metadata, showQrModal: false }))
```

Comment:
Do we use `alphaWalletConnect` in this file somewhere else? Seems still imported, prob it'll throw error in the base branch by linter

Reply:
ty, removed